blocks/persistence/exceptions.h

```diff
@@ -64,6 +64,12 @@ struct PersistenceFileNotWritable : PersistenceException {
       : PersistenceException("Persistence file not writable: `" + filename + "`.") {}
 };
 
+struct UnsafePublishBadIndexTimestampException : PersistenceException {
+  explicit UnsafePublishBadIndexTimestampException(uint64_t expected, uint64_t found)
+      : PersistenceException(current::strings::Printf(
+          "Expecting index %lld, seeing %lld.", static_cast<long long>(expected), static_cast<long long>(found))) {}
```

Comment:
BTW, cross-platform macro for printing `uint64_t` is `PRIu64`

Reply:
We have none of those in Current btw, and I kind of like the pure way with `%lld` and a `static_cast<long long>`. Btw, `3rdparty/rapidjson/msinttypes/inttypes.h` [re]defines this macro for some old Visual Studio builds.